modules/DefaultRenderer.js

```diff
@@ -5,7 +5,7 @@ import React, { Component } from 'react'
 import { NativeModules, StyleSheet, Platform, View } from 'react-native'
 import { Transitioner } from 'react-navigation'
 import Card from 'react-navigation/src/views/Card'
-import CardStackPanResponder from 'react-navigation/src/views/CardStackPanResponder'
+import CardStackStyleInterpolator from 'react-navigation/src/views/CardStackStyleInterpolator'
```

Comment:
Please, remove this changes. ```react-navigation@1.0.0-beta.8``` will be support in [next branch](https://github.com/LeoLeBras/react-router-navigation/tree/next)